pyexasol/connection.py

```diff
@@ -203,16 +285,54 @@ def __init__(self
     def execute(self, query, query_params=None) -> ExaStatement:
         """
         Execute SQL query with optional query formatting parameters
-        Return ExaStatement object
+
+        Args:
+            query:
+                SQL query text, possibly with placeholders
+            query_params:
+                 Values for placeholders
+
+        Returns:
+            ExaStatement object
+
+        Examples:
+
+            >>> con = ExaConnection(...)
+            >>> con.execute(
+            ...        query="SELECT * FROM {table!i} WHERE col1={col1}",
+            ...        query_params={'table': 'users', 'col1':'bar'}
+            ...)
         """
         return self.cls_statement(self, query, query_params)
 
     def execute_udf_output(self, query, query_params=None):
         """
         Execute SQL query with UDF script, capture output
-        Return ExaStatement object and list of Path-objects for script output log files
 
-        Exasol should be able to open connection to the machine where current script is running
+        Note:
+            Exasol should be able to open connection to the machine where current script is running.
+            It is usually OK in the same data centre, but it is normally not working
+            if you try to run this function on local laptop.
+
+        Args:
+            query:
+                SQL query text, possibly with placeholders
+            query_params:
+                Values for placeholders |
+
+        Returns:
+            Return tuple with two elements: (1) instance of :class:`pyexasol.ExaStatement`
+            and (2) list of :class:`Path` objects for script output log files.
+
+        Attention:
+            Exasol should be able to open connection to the machine where current script is running
```

Comment:
Same as above

Reply:
same as above, also happy to discuss this in person real quick ;)